lighthouse-core/gather/driver.js

```diff
@@ -187,9 +187,10 @@ class Driver {
    * Call protocol methods
    * @param {!string} method
    * @param {!Object} params
+   * @param {boolean=} silent
    * @return {!Promise}
    */
-  sendCommand(method, params) {
+  sendCommand(method, params, silent) {
```

Comment:
the boolean trap is pretty gnarly. Before we hardcoded some error message to ignore it...we could do the same again. There could also be some kind of `log` trickery so that this call to `sendCommand` could silence an error it was expecting then turn the silencing off again. Pretty hacky ideas, though...